patches/server/0929-Add-Entity-Body-Yaw-API.patch

```diff
@@ -4,6 +4,42 @@ Date: Thu, 2 Feb 2023 16:40:41 +0100
 Subject: [PATCH] Add Entity Body Yaw API
 
 
+diff --git a/src/main/java/org/bukkit/craftbukkit/entity/CraftEntity.java b/src/main/java/org/bukkit/craftbukkit/entity/CraftEntity.java
+index 733158b6f2c2bd03fbe798562ff7bc33280548dc..64afd57cd2f8db802c4ec5ca83b07bd5fd887ed1 100644
+--- a/src/main/java/org/bukkit/craftbukkit/entity/CraftEntity.java
++++ b/src/main/java/org/bukkit/craftbukkit/entity/CraftEntity.java
+@@ -1419,6 +1419,31 @@ public abstract class CraftEntity implements org.bukkit.entity.Entity {
+     public boolean isInPowderedSnow() {
+         return getHandle().isInPowderSnow || getHandle().wasInPowderSnow; // depending on the location in the entity "tick" either could be needed.
+     }
++
++    @Override
++    public double getX() {
++        return this.entity.getX();
++    }
++
++    @Override
++    public double getY() {
++        return this.entity.getY();
++    }
++
++    @Override
++    public double getZ() {
++        return this.entity.getZ();
++    }
++
++    @Override
++    public float getPitch() {
++        return this.entity.getXRot();
++    }
++
++    @Override
++    public float getYaw() {
++        return this.entity.getYRot();
```

Comment:
This will not return the bukkit yaw and will not match #getLocation, you can see the bukkit implementation of #getLocation to behave the same for living entities

Reply:
Should correctly use #getBukkitYaw now.